examples/monitor-with-thanos/README.md

```diff
@@ -0,0 +1,48 @@
+# TiDBMonitor with Thanos
+
+> **Note:**
+>
+> This document is to show how to integrate tidbmonitor into [thanos](https://thanos.io/design.md/).
+>
+The following steps will create a TiDB cluster by default.
+
+
+## Install
+
+The following commands is assumed to be executed in this directory.
+
+Install the monitor with thanos sidecar:
+
+```bash
+> kubectl -n <namespace> apply -f tidb-monitor.yaml
+```
+
+Wait for monitor Pods ready:
+
+```bash
+watch kubectl -n <namespace> get pod
+```
+
+If you need to store historical data,you can spec `objectStorageConfig` field and create the corresponding secret.
+
+## Install Thanos
+
+Install thanos query component to integrate tidbmonitor :
+
+```bash
+> kubectl -n <namespace> apply -f thanos-query.yaml
```

Comment:
We should combine the svc and deployment yaml into one file?

Reply:
Ok.